### DISCUSSION TITLE
Оптимизация процессов по CSV-импорту / ролевой модели

### DISCUSSION THREAD

**Discussion by Jeqsuz0** [totumonline/totum-ru-issues-and-discussions#629] [Как это сделать?]
Добрый день, есть пару вопросов:
1. Расскажите, пожалуйста, как можно оптимизировать функцию CSV-импорта в расчётную таблицу циклов? Нужно загрузить 30к+ строк, не загружает даже 500 строк, уже увеличили время обработки запроса, пробовали через postman, через remotes, также не вышло (+ выключали все функции обработки атрибутов, чтобы снизить нагрузку)
2. Можно ли как-то настроить УЗ для Создателя с ограниченными возможностями? А именно, нужна УЗ для настройки ролевой модели на платформе, но с ограничением тем, что нельзя этой УЗ создавать/редактировать/удалять формы и атрибуты + менять другие параметры, которые есть у создателя, а только можно настраивать роли

**Comment by erpic-ru** [comment]
1. Создаете простую таблицу, грузите в нее все, а потом кодами создаете нужные справочники, а также создаете и заполняете циклы. 

2. Можно дать в доступ на редактирование таблицу ролей, например
